.travis.yml

```diff
@@ -1,11 +1,16 @@
 language: java
 jdk:
-  - oraclejdk8
+- oraclejdk8
 sudo: required
 env:
-  - HUDI_QUIETER_LOGGING=1
+- HUDI_QUIETER_LOGGING=1
 services:
-  - docker
+- docker
 cache:
   directories:
-  - $HOME/.m2
+  - "$HOME/.m2"
```

Comment:
indentation of this seems to have changed?

Reply:
It was auto generated by the command (https://docs.travis-ci.com/user/notifications/)
travis encrypt "<account>:<token>#channel" --add notifications.slack.rooms

I can revert other changes manually if preferred.